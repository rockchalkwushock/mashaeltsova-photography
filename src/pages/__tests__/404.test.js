import React from 'react'
import { mount, shallow } from 'enzyme'

import NotFound from '../404'

describe('Pages: 404', () => {
  test('should render markup', () => {
    const wrapper = mount(<NotFound />)
    expect(wrapper).toMatchSnapshot()
  })
  test('should have props', () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper.find('Title').props()).toHaveProperty('color', 'slategrey')
    expect(wrapper.find('Title').props()).toHaveProperty('text', 'NOT FOUND')
  })
})
