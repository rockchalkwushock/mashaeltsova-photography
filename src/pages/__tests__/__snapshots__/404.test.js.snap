// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages: 404 should render markup 1`] = `
.c0 {
  color: slategrey;
}

<NotFoundPage>
  <div>
    <Title
      color="slategrey"
      text="NOT FOUND"
    >
      <StyledTitle
        color="slategrey"
      >
        <h1
          className="c0"
          color="slategrey"
        >
          NOT FOUND
        </h1>
      </StyledTitle>
    </Title>
  </div>
</NotFoundPage>
`;
